### DISCUSSION TITLE
Copilot suggests an off-topic sexual invitation

### DISCUSSION THREAD

**Discussion by migliori** [community/community#125810] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

In vs -code, I start to write this simple text (in French, but I give the English translation here:

> Hello everyone,

Copilot generates the following (that's real ...):

> I'm a 25-year-old man, and I've been in a relationship for 5 years with a 24-year-old woman. We have a very active sex life, but for some time now I've been wanting to have sex with a man. I've never had a relationship with a man, but I'm keen to find out more about this experience. I'm looking for a man who could introduce me to sex between men. I'm open to all propositions, I'm very curious and I want to discover new things. I'm a very discreet man, I'm in a relationship and I don't want my girlfriend to know about my desires. If you're interested, don't hesitate to contact me. I am open to all proposals. See you soon.

**Comment by denisdasilvarocha** [comment]
Screen or didnt happend

**Reply by migliori** [reply]
Here it is:

![Capture d’écran 2024-05-26 201801](https://github.com/community/community/assets/7558100/14426ed1-faa7-4909-bcf5-a54221db373c)

**Comment by ettaboyle** [comment]
Hi @migliori, Thank you for your report. We've shared this with the appropriate team internally and they are investigating this.
